client/src/components/ManageArtist/ManageArtistContainer.tsx

```diff
@@ -10,7 +10,7 @@ import { useArtistContext } from "state/ArtistContext";
 
 const Container = styled.div<{ artistBanner: boolean; userId?: number }>`
   width: 100%;
-  margin-top: calc(16vh);
+  margin-top: calc(12vh);
```

Comment:
I feel like this is changing on every PR 😆

Reply:
Oh on this one it just changed because the header became transparent on artist page ! so it left us more margin to visualize more of the artist page at first glance.